src/wrappers/themis/react-native-themis/src/index.js

```diff
@@ -19,6 +19,18 @@ export function isBase64(str) {
 export function string64(input) {
     return Buffer.from(input).toString('base64');
 }
+const convertInputBase64 = (param, name, base64) => {
+    if (param === "" || param === undefined || param === null) {
+        throw new Error(`Parameter ${name} can not be empty`);
+    }
+    if (base64) {
+        if (!isBase64(param)) {
+            throw new Error(`Parameter ${name} is not base64 encoded`);
+        }
+        return Array.from(Buffer.from(param, 'base64')); // return Uint8Array from base64
+    }
+    return param;
+};
```

Comment:
I tend to find boolean parameters bad for readability. IMO, this will be more readable with two separate helper functions: `convertInputBase64` that does the conversion, and, for example, `convertInput` that does not.

Reply:
done.